services/has-many-getter.js

```diff
@@ -117,7 +113,17 @@ function HasManyGetter(model, association, opts, params) {
           fieldsSearched = searchBuilder.getFieldsSearched();
         }
 
-        return [records, count, fieldsSearched];
+        records = _.slice(records, getSkip(), getSkip() + getLimit());
+
+        return [records, fieldsSearched];
+      });
+  };
+
+  this.count = function () {
+    return getRecords()
+      .then(function (records) {
+        count = records.length;
```

Comment:
count is defined somewhere?

Reply:
It was unnecessary defined globally, fixed, thanks 👍